dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/DhisAnalyticCharts.kt

```diff
@@ -11,10 +11,22 @@ class DhisAnalyticCharts @Inject constructor(
         return true
     }
 
-    override fun getCharts(enrollmentUid: String): List<Graph> {
+    override fun geEnrollmentCharts(enrollmentUid: String): List<Graph> {
         return chartsRepository.getAnalyticsForEnrollment(enrollmentUid)
     }
 
+    override fun getProgramVisualizations(programUid: String): List<Graph> {
+        return chartsRepository.getAnalyticsForProgram(programUid)
+    }
+
+    override fun getHomeVisualizations(): List<Graph> {
```

Comment:
if we don't need this right now, why don't we declare them when needed?

Reply:
The idea was to have these charts (Pie and Radar) ready to be used by the visualizations in the search, home and dataSet screens... but we run into a blocker in the SDK (missing objects). They can be removed yes.